CHANGELOG.md

```diff
@@ -1,5 +1,200 @@
+- Decide if the start method of Publisher and Subscriber SDKs should be async [\#117](https://github.com/ably/ably-asset-tracking-swift/issues/117)
+- Subscriber should be able to request no resolution after creation [\#116](https://github.com/ably/ably-asset-tracking-swift/issues/116)
+- Remove empty amplifyconfiguration.json file. [\#114](https://github.com/ably/ably-asset-tracking-swift/issues/114)
+- Handle resolutions for the location engine in the Publisher [\#113](https://github.com/ably/ably-asset-tracking-swift/issues/113)
+- Change stop method in Subscriber SDK to an asynchronous one [\#108](https://github.com/ably/ably-asset-tracking-swift/issues/108)
+- Unify wrappers for the Ably SDK [\#107](https://github.com/ably/ably-asset-tracking-swift/issues/107)
+- Update Mapbox dependency to Nav 2.0 [\#106](https://github.com/ably/ably-asset-tracking-swift/issues/106)
+- Investigate use of publisherService didFailWithError [\#103](https://github.com/ably/ably-asset-tracking-swift/issues/103)
+- Better approach for Objective-C facade [\#102](https://github.com/ably/ably-asset-tracking-swift/issues/102)
+- Check ErrorInformation alignment with Android codebase [\#101](https://github.com/ably/ably-asset-tracking-swift/issues/101)
+- Unify the way of parsing enums sent over Ably channels across all platforms [\#99](https://github.com/ably/ably-asset-tracking-swift/issues/99)
+- Trackables should be comparable by id, not by all fields [\#92](https://github.com/ably/ably-asset-tracking-swift/issues/92)
+- Reflect tracked asset state in publisher example app [\#89](https://github.com/ably/ably-asset-tracking-swift/issues/89)
+- iOS: Send elevation and battery level of the device as part of the update [\#84](https://github.com/ably/ably-asset-tracking-swift/issues/84)
+- Clear and visible UI indications onSuccess and onFailure [\#76](https://github.com/ably/ably-asset-tracking-swift/issues/76)
+- Create a project with snippets of Objective-C code calling our SDKs [\#74](https://github.com/ably/ably-asset-tracking-swift/issues/74)
+- iOS: Remove sending raw updates from the publisher SDK [\#73](https://github.com/ably/ably-asset-tracking-swift/issues/73)
+- Use AssetTrackingError where possible [\#66](https://github.com/ably/ably-asset-tracking-swift/issues/66)
+- Create ARTRealtime instances with `clientId` parameter [\#65](https://github.com/ably/ably-asset-tracking-swift/issues/65)
+- Adopt callback interface changes from Android - single handler, not discrete onSuccess and onError [\#64](https://github.com/ably/ably-asset-tracking-swift/issues/64)
+- Accept new `LocationUpdate` format on the Subscriber side [\#63](https://github.com/ably/ably-asset-tracking-swift/issues/63)
+- Handle `Publisher.stop()` correctly [\#51](https://github.com/ably/ably-asset-tracking-swift/issues/51)
+- Find a better way of importing sources from Core to Publisher and Subscriber [\#43](https://github.com/ably/ably-asset-tracking-swift/issues/43)
```

Comment:
This feels like something that does not carry any value for the SDK user / app developer. I would remove it.

Reply:
Removed in [e2cbd82](https://github.com/ably/ably-asset-tracking-swift/pull/322/commits/e2cbd82ad81e7780ba1e00d6cf4f5e581768340d). I also made another pass over this file and removed some additional entries.